target/linux/lantiq/image/Makefile

```diff
@@ -560,42 +595,50 @@ TARGET_DEVICES += zyxel_p-2812hnu-f1
 
 define Device/zyxel_p-2812hnu-f3
   $(Device/NAND)
+  DEVICE_VENDOR := ZyXEL
+  DEVICE_MODEL := P-2812HNU
+  DEVICE_VARIANT := F3
   BOARD_NAME := P2812HNUF3
   DEVICE_DTS := P2812HNUF3
-  DEVICE_TITLE := ZyXEL P-2812HNU-F3
   DEVICE_PACKAGES := kmod-rt2800-pci wpad-basic kmod-usb-dwc2
   SUPPORTED_DEVICES += P2812HNUF3
 endef
 TARGET_DEVICES += zyxel_p-2812hnu-f3
 
 define Device/arcadyan_arv7519rw22
+  DEVICE_VENDOR := Livebox
+  DEVICE_MODEL := Astoria ARV7519RW22
```

Comment:
Hmm, that might be `Astoria (ARV7519RW22)` if we follow the style from above.

Reply:
Astoria Networks is apparently a brand of Arcadyan, see also
https://openwrt.org/toh/hwdata/arcadyanastoria/start
Hence I suggest
```
DEVICE_VENDOR := Arcadyan
DEVICE_MODEL := ARV7519RW22
DEVICE_ALT0_VENDOR := Astoria
DEVICE_ALT0_MODEL := Livebox
```

Reply:
From my point of view, it would really make sense to tackle all of the devices with this concept already in this PR.